scripts/interactive_tests/interactive_test_util.js

```diff
@@ -8,7 +8,12 @@ var TIMEOUT = 10000;
 var Server = function(serverStartCmd, port) {
   // Start protractor and its debugger server as a child process.
   this.start = function() {
-    var deferred = q.defer();
+    let deferredResolve;
```

Comment:
Similar to my comments below, we should just return a promise instead of using a deferred here